### PR TITLE
Add ES translations

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚 Description

Add the Spanish translations to all `strings.xml`

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="post_not_found">Post no encontrado</string>
+    <string name="channel_image">Imagen del canal</string>
+    <string name="referenced_event_not_found">Evento referenciado no encontrado</string>
+    <string name="could_not_decrypt_the_message">No se pudo desencriptar el mensaje</string>
+    <string name="group_picture">Imagen de grupo</string>
+    <string name="explicit_content">Contenido explícito</string>
+    <string name="spam">Spam</string>
+    <string name="impersonation">Suplantación de identidad</string>
+    <string name="illegal_behavior">Comportamiento ilegal</string>
+    <string name="unknown">Desconodido</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:20]:
```suggestion
    <string name="unknown">Desconocido</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="relay_icon">Icono de relé</string>
+    <string name="unknown_author">Autor desconocido</string>
+    <string name="copy_text">Copiar texto</string>
+    <string name="copy_user_pubkey">Copiar PubKey del usuario</string>
+    <string name="copy_note_id">Copiar ID de la nota</string>
+    <string name="broadcast">Transmisión</string>
+    <string name="block_hide_user">Bloquear y ocultar usuario></string>
+    <string name="report_spam_scam">Reportar Spam / Estafa</string>
+    <string name="report_impersonation">Reportar suplantación de identidad</string>
+    <string name="report_explicit_content">Reportar contenido explicito</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:30]:
```suggestion
    <string name="report_explicit_content">Reportar contenido explícito</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="send_sats">Enviar Sats</string>
+    <string name="never_translate_from">"Nunca traducir desde "</string>
+    <string name="error_parsing_preview_for">"Error al analizar la vista previa de %1$s : %2$s"</string>
+    <string name="preview_card_image_for">"Imagen de vista previa para %1$s"</string>
+    <string name="new_channel">Nuevo canal</string>
+    <string name="channel_name">Nombre del canal</string>
+    <string name="my_awesome_group">Mi grupo genial</string>
+    <string name="picture_url">URL de imagen</string>
+    <string name="description">Descripción</string>
+    <string name="about_us">"Sobre nosotros.. "</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:68]:
```suggestion
    <string name="about_us">"Sobre nosotros... "</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="cancel">Cancelar</string>
+    <string name="failed_to_upload_the_image">No se pudo cargar la imagen</string>
+    <string name="relay_address">Dirección del transmisor</string>
+    <string name="posts">Publicaciones</string>
+    <string name="errors">Errores</string>
+    <string name="home_feed">Tus noticias</string>
+    <string name="private_message_feed">Fuente de mensajes privados</string>
+    <string name="public_chat_feed">Fuente de chat público</string>
+    <string name="global_feed">Fuente global</string>
+    <string name="add_a_relay">Anadir tramsmisor</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:82]:
```suggestion
    <string name="add_a_relay">Anadir transmisor</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="channel_image">Imagen del canal</string>
+    <string name="referenced_event_not_found">Evento referenciado no encontrado</string>
+    <string name="could_not_decrypt_the_message">No se pudo desencriptar el mensaje</string>
+    <string name="group_picture">Imagen de grupo</string>
+    <string name="explicit_content">Contenido explícito</string>
+    <string name="spam">Spam</string>
+    <string name="impersonation">Suplantación de identidad</string>
+    <string name="illegal_behavior">Comportamiento ilegal</string>
+    <string name="unknown">Desconodido</string>
+    <string name="relay_icon">Icono de relé</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:21]:
```suggestion
    <string name="relay_icon">Icono de transmisión</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="display_name">Nombre para mostrar</string>
+    <string name="my_display_name">Mi nombre para mostrar</string>
+    <string name="username">Nombre de usuario</string>
+    <string name="my_username">Mi nombre de usuario</string>
+    <string name="about_me">Sobre mí</string>
+    <string name="avatar_url">URL del avatar</string>
+    <string name="banner_url">URL del banner</string>
+    <string name="website_url">Dirección web</string>
+    <string name="ln_address">Dirección LN</string>
+    <string name="ln_url_outdated">Dirección LN (outdated)</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:92]:
```suggestion
    <string name="ln_url_outdated">Dirección LN (antiguo)</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="image_saved_to_the_gallery">Imagen guardada en la galería</string>
+    <string name="failed_to_save_the_image">Error al guardar la imagen</string>
+    <string name="upload_image">Cargar imagen</string>
+    <string name="uploading">Cargando…</string>
+    <string name="user_does_not_have_a_lightning_address_setup_to_receive_sats">El usuario no tiene una configuración de dirección Lightning para recibir sats</string>
+    <string name="reply_here">"responde aquí.. "</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:98]:
```suggestion
    <string name="reply_here">"responde aquí... "</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="new_requests">Nuevas solicitudes</string>
+    <string name="blocked_users">Usuarios bloqueados</string>
+    <string name="new_threads">Temas nuevos</string>
+    <string name="conversations">Conversaciones</string>
+    <string name="notes">Notas</string>
+    <string name="replies">Respuestas</string>
+    <string name="follows">"Sigue"</string>
+    <string name="reports">"Reportes"</string>
+    <string name="more_options">Más opciones</string>
+    <string name="relays">" Retrasmisores"</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:113]:
```suggestion
    <string name="relays">" Retransmisores"</string>
```

File: app/src/main/res/values-es/strings.xml [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+    <string name="unblock_user">Desbloquear usuario</string>
+    <string name="npub_hex_username">"npub, hex, username "</string>
+    <string name="clear">Limpiar</string>
+    <string name="app_logo">Logo de la app</string>
+    <string name="nsec_npub_hex_private_key">nsec / npub / hex private key</string>
+    <string name="show_password">Mostrar contraseña</string>
+    <string name="hide_password">Ocultar contraseña</string>
+    <string name="invalid_key">Clave invalida</string>
+    <string name="i_accept_the">"Acepto los "</string>
+    <string name="terms_of_use">terminos de uso</string>
```
Comment (JesusValera) [inline app/src/main/res/values-es/strings.xml:134]:
```suggestion
    <string name="terms_of_use">términos de uso</string>
```

**Review by JesusValera — COMMENTED** [review]
Some typos :octocat:

**Review by JesusValera — APPROVED** [review]

**Comment by vitorpamplona** [comment]
@Chemaclass and @JesusValera what are your npubs?

**Comment by Chemaclass** [comment]
Mine: npub1ch2gzhpxux8zc9upxvqy5mwm4x5k5hm6772682mqd5g65yz4z34qg4sxwx

**Comment by JesusValera** [comment]
Here is mine: `npub1de6xwq8zprgcp9gx86d53zu5pa8mu43ze7cmqvgkvzh6srm5vfqsz7ye0x` ([snort](https://snort.social/p/npub1de6xwq8zprgcp9gx86d53zu5pa8mu43ze7cmqvgkvzh6srm5vfqsz7ye0x))

**Pull request merged**